zeppelin-web/src/app/jobmanager/jobs/job.controller.js

```diff
@@ -37,6 +37,68 @@
       var result = Math.ceil(runningJobCount / totalCount * 100);
       return isNaN(result) ? 0 : result;
     };
+
+    $scope.runNotebookJob = function(notebookId) {
+      BootstrapDialog.confirm({
+        closable: true,
+        title: '',
+        message: 'Run all paragraphs?',
+        callback: function(result) {
+          if (result === true) {
```

Comment:
could you please change it to `if (result)` here and elsewhere?
https://github.com/airbnb/javascript#comparison--shortcuts